rules/ml/ml_linux_anomalous_metadata_process.toml

```diff
@@ -19,7 +21,7 @@ false_positives = [
 from = "now-45m"
 interval = "15m"
 license = "Elastic License v2"
-machine_learning_job_id = ["linux_rare_metadata_process", "v2_linux_rare_metadata_process"]
+machine_learning_job_id = ["v3_linux_rare_metadata_process"]
```

Comment:
Why aren't we appending the v3 jobs to the list like we did v2? Does this mean v3 jobs are the only supported ML jobs moving forward?

Reply:
yes the v1 and v2 jobs are gone, they were written for older schemas and it was confusing having to use one or both modules depending on the versions of things in the mix in a given deployment.